articles/api/v2/query-string-syntax.md

```diff
@@ -0,0 +1,166 @@
+# Query String Syntax
+
+This page will help you to learn query string syntax, the "mini-language" used by the Query String Query. This will help to form your own queries in the `q` query string parameter when searching using Auth0's [API](/api/v2).
+
+The following information is adapted from [Elastic's](elastic.co) Elasticsearch Reference.
+
+The query string is parsed into a series of *terms* and *operators*. A term can be a single word — `quick` or `brown` — or a phrase, surrounded by double quotes — `"quick brown"` — which searches for all the words in the phrase, in the same order.
+
+### Field Names
+
+* where the `date` field contains `2016`
+
+`date:2016`
+
+* where the `user_name` field contains `john` or `smith`. If you omit the OR operator the default operator will be used. 
+
+`user_name: (john OR smith`
+`user_name: (john smith)`
+
+* where the `user_name` filed contains the exact phrase `"john smith"`
+
+`user_name: "john smith"`
+* where any of the fields of `app_metadata` contain `john` or `smith` (note how we need to escape the * with a backslash)
+
+`app_metadata.\*: (john smith)`
+
+* where the field `description` has no value (or is missing)
+
+`_missing_: description`
+
+* where the field `description` has any non-null value
+
+`_exists_: description`
+
+### Wildcards
+Wildcard searches can be run on individual terms, using `?` to replace a single character, and `*` to replace zero or more characters:
+
+`2016-0?-*`
+
+Be aware that wildcard queries can use an enormous amount of memory and perform very badly — just think how many terms need to be queried to match the query string `"a* b* c*"`.
+
+::: panel-warning Warning
+
+Allowing a wildcard at the beginning of a word (eg "\*ing") is particularly heavy, because all terms in the index need to be examined, just in case they match. Leading wildcards can be disabled by setting allow_leading_wildcard to false.
+:::
+
+Wildcarded terms are not analyzed by default — they are lowercased (`lowercase_expanded_terms` defaults to `true`) but no further analysis is done, mainly because it is impossible to accurately analyze a word that is missing some of its letters. However, by setting `analyze_wildcard` to true, an attempt will be made to analyze wildcarded words before searching the term list for matching terms.
```

Comment:
the lowercase_expanded_terms sounds like an elasticsearch setting that the user should not have to know about